src/core/stream.js

```diff
@@ -1073,21 +1076,12 @@ var Jbig2Stream = (function Jbig2StreamClosure() {
     var jbig2Image = new Jbig2Image();
 
     var chunks = [];
-    var decodeParams = this.dict.getArray('DecodeParms', 'DP');
-
-    // According to the PDF specification, DecodeParms can be either
-    // a dictionary, or an array whose elements are dictionaries.
-    if (isArray(decodeParams)) {
-      if (decodeParams.length > 1) {
-        warn('JBIG2 - \'DecodeParms\' array with multiple elements ' +
```

Comment:
This warning seems to have been removed in the new version. What is the reason for that? I think if it's something that we erroneously don't support that we should still issue a warning to make debugging easier.